wgpu-native/src/device.rs

```diff
@@ -1020,15 +1023,13 @@ pub fn device_create_bind_group(
             binding_model::BindingResource::Buffer(ref bb) => {
                 let (alignment, usage) = match decl.ty {
                     binding_model::BindingType::UniformBuffer
-                    | binding_model::BindingType::UniformBufferDynamic => (
-                        device.limits.min_uniform_buffer_offset_alignment,
-                        resource::BufferUsage::UNIFORM,
-                    ),
+                    | binding_model::BindingType::UniformBufferDynamic => {
+                        (DYNAMIC_BUFFER_OFFSET_ALIGN, resource::BufferUsage::UNIFORM)
```

Comment:
we should definitely rename it from DYNAMIC_, perhaps to `DESCRIPTOR_BUFFER_OFFSET_ALIGN`?